src/Connectors/test/Connector.EFCore.Test/Steeltoe.Connector.EFCore.Test.csproj

```diff
@@ -17,12 +17,8 @@
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="$(EFCoreTestVersion)" />
     <PackageReference Include="Oracle.EntityFrameworkCore" Version="$(EFCoreOracleTestVersion)" />
   </ItemGroup>
-  <!-- Come back to this in the future with another strategy
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+  <ItemGroup>
```

Comment:
There's no more need to have separate `ItemGroup`s containing `PackageReference`s now, they can be merged into one.